getting-started/templates/systemlink-values.yaml

```diff
@@ -306,6 +306,60 @@ minio:
 ## Configuration for test result storage.
 ##
 testmonitorservice:
+  ## Configuring additional sidecars for the Test Monitor Service.
+  ##
+  ## @param sidecars Additional containers to run in the same pod as the Test Monitor Service.
+  ## It is used for authentication proxy containers, such as the Cloud SQL Auth Proxy. Example of a Cloud SQL Auth Proxy sidecar has been provided below.
+  ## ref: https://cloud.google.com/sql/docs/postgres/connect-kubernetes-engine#run_the_in_a_sidecar_pattern
+  # sidecars:
+  #   - name: cloud-sql-proxy
```

Comment:
Example commented values is not needed in the values template. Just the documentation and reference link is enough.

We have [supplement ](https://github.com/ni/install-systemlink-enterprise/tree/main/getting-started/templates/AWS) values for AWS and Azure, we can include the example there similarly for GCP

Reply:
uncomment the fields `sidecars` and `extraVolumes`

Reply:
The addition of supplemental values for gcp can be taken up in a follow up PR